app/models/shelter.rb

```diff
@@ -31,4 +31,12 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where("age >= ?", age_filter)
   end
+
+  def self.order_desc_alphabetical
+    find_by_sql("SELECT * FROM shelters ORDER BY name DESC")
+  end
+
+  def self.pending_applications
+    joins(pets: :applications).where("applications.status = 'Pending'").distinct
```

Comment:
Great work! This is a nice, concise joins method. 

Another nifty option: Allow Shelter to mooch off the associations that Pet has set up. The Pet model has the associations: `has_many :applications, through: pet_applications`. You can build off of associations in Rails, so in your Shelter model, because Shelter `has_many :pets`, you can also add `has_many :applications, through: :pets`. Then, you can access `applications` as an attribute of a shelter! It will do this multi-part joining to the applications table under the hood!